canonical/ubuntu.com
# Ibm juju homepage

## Done

Build juju take under. Build 16.04 Takeover - cloud - IBM 
## QA

Yaili has reviewed and signed off both.

Go to the homepage and check the takeover and takeunder at s/m and l and make sure the code is groovy
## Issue / Card

Takeover card: https://trello.com/c/XIgLlYV3
Takeover doc: https://docs.google.com/document/d/1M8ZrU6mW_6YiRogRo3oqSPIO2wa9T_URMkKaCNbnfJM/edit?ts=56fe6b57#

Takeunder card: https://trello.com/c/k5uNJ6s2
Takeunder doc: https://docs.google.com/document/d/17CVb7b3KaNuKQ3LNMhyxxFmus-ohpPnjGghs65-yEBI/edit
